packages/picasso.js/package.json

```diff
@@ -24,6 +24,7 @@
   ],
   "main": "dist/picasso.js",
   "module": "src/index.js",
+  "source": true,
```

Comment:
This is useful if you use Parcel to build a project that you `npm link` to picasso.js ... zero config ^^